packages/terra-date-picker/src/DateUtil.js

```diff
@@ -28,6 +28,16 @@ class DateUtil {
 
     return momentDates.length > 0 ? momentDates : dates;
   }
+
+  // Converts date string to the ISO8601 format in UTC. If the date string is invalid and unable to convert, the originally provided string is returned.
+  static convertToISO8601(date, format) {
```

Comment:
That sounds fair. I take it then that if the function is successful that it will never return the same value that was passed in. Since you documented that approach, it should be fine.